lib/telegraf/telegraf.conf.erb

```diff
@@ -616,6 +619,7 @@
   #
   ## Some queries we may want to run less often (such as SHOW GLOBAL VARIABLES)
   interval_slow                   = "30m"
```

Comment:
Do we know if any of the things we're passing are in the slow category?  I don't think so, but if they are, we may need to make this interval smaller.